openslides_backend/action/actions/organization/initial_import.py

```diff
@@ -75,6 +79,27 @@ def check_empty_datastore(self) -> None:
         ):
             raise ActionException("Datastore is not empty.")
 
+    def check_migration_index(self, instance: Dict[str, Any]) -> None:
+        data_migration_index = instance["data"].get("_migration_index")
+        if data_migration_index is None:
+            raise ActionException(
+                "Data must have a valid migration index in `_migration_index`."
+            )
+        if data_migration_index < 1:
+            raise ActionException(
+                f"Data must have a valid migration index >= 1, bus has {data_migration_index}."
+            )
+
+        backend_migration_index = get_backend_migration_index()
+        if backend_migration_index < data_migration_index:
+            raise ActionException(
+                f"Migration indices do not match: Data has {data_migration_index} and the backend has {backend_migration_index}"
+            )
+        self.data_migration_index = data_migration_index
+        self.migration_needed = False
+        if backend_migration_index > data_migration_index:
+            self.migration_needed = True
```

Comment:
Does this flag do anything? Unless you want to migrate on the fly during the initial import (which I'm against) I would argue that only the current MI can be imported. In practice, this will always be the case anyway - we always keep our initial/example data up-to-date.

Reply:
Another advantage: You can remove the whole MI handling from the action code again and just set it below in the `create_write_requests`. It is invalid anyway for any action except `initial_import` to set a migration index, so we should not give all actions possibility to do so. See below for details

Reply:
I think we should migrate "on the fly"! Even better, the need to migrate should be checked by the starting procedure. 
Reason: It is not guaranteed that the initial import has the correct MI. Imagine a customized initial-data file and the latest version will  be or is deployed. Nobody wants to do the migration by hand, but it is easy for us implement such a logic.
Let's talk about it tomorrow.
That's the reason for the self.migration_needed flag.

> > Couldn't we set mediafile.is_public = True to all mediafiles, except the meeting one's?
> 
> Sure, would be okay for me.



> > Couldn't we set mediafile.is_public = True to all mediafiles, except the meeting one's?
> 
> Sure, would be okay for me.

Including setting the field required?